opencog/reasoning/pln/rules/inheritance-to-member.scm

```diff
@@ -0,0 +1,51 @@
+; =============================================================================
+; InheritanceToMemberRule
+;	InheritanceLink( B C )
+;			becomes
+;	MemberLink( B C )
+; -----------------------------------------------------------------------------
+
```

Comment:
@prateeksaxena2809 I don't to which extend SetLink is supported but that would be the right thing to do.